src/hearings/containers/hearing-actuals/hearing-actuals-view-edit-parties/hearing-actuals-view-edit-parties.component.ts

```diff
@@ -169,10 +169,9 @@ export class HearingActualsViewEditPartiesComponent implements OnInit, OnDestroy
   private createForm(hearingActualsMainModel: HearingActualsMainModel): void {
     const actualDayIndex = ActualHearingsUtils.getActualDayIndexFromHearingDate(hearingActualsMainModel, this.hearingDate);
 
-    const hasActualParties = hearingActualsMainModel.hearingActuals && hearingActualsMainModel.hearingActuals.actualHearingDays
-      && hearingActualsMainModel.hearingActuals.actualHearingDays.length && hearingActualsMainModel.hearingActuals.actualHearingDays[actualDayIndex]
-      && hearingActualsMainModel.hearingActuals.actualHearingDays[actualDayIndex].actualDayParties
-      && hearingActualsMainModel.hearingActuals.actualHearingDays[actualDayIndex].actualDayParties.length > 0;
+    const hasActualParties = hearingActualsMainModel?.hearingActuals?.actualHearingDays?.length
```

Comment:
@johnbenjamin-hmcts - does this still work if `actualHearingDays?.length` is `0`, remembering that `0` is falsy?

Reply:
@Daniel-Lam - yes, if `0` then participants are chosen from the planned model.

At the beginning of the process, the hearingActuals will be always null and the participants and timings will be copied from Planned to Actuals when the user confirms. So, if the actuals object is not null, then it will definitely contain at least one actualHearingDays.